internal/servers/controller/handlers/authmethods/authmethod_service.go

```diff
@@ -260,6 +262,28 @@ func (s Service) Authenticate(ctx context.Context, req *pbs.AuthenticateRequest)
 	return &pbs.AuthenticateResponse{Item: tok, TokenType: req.GetTokenType()}, nil
 }
 
+// AuthenticateLogin implements the interface pbs.AuthenticationServiceServer.
+func (s Service) AuthenticateLogin(ctx context.Context, req *pbs.AuthenticateLoginRequest) (*pbs.AuthenticateLoginResponse, error) {
+	if err := validateAuthenticateLoginRequest(req); err != nil {
+		return nil, err
+	}
+	authResults := s.authResult(ctx, req.GetAuthMethodId(), action.Authenticate)
```

Comment:
Should this be a subaction now or do we want to wait to add subactions when we expect people to actually want to grant permissions based on those subactions?

Reply:
We don't currently plant to make these separately permissioned subactions.